lib/matplotlib/tests/test_transforms.py

```diff
@@ -219,8 +217,8 @@ def transform_path_non_affine(self, path):
         return self.real_trans.transform_path(path)
 
 
-class BasicTransformTests(unittest.TestCase):
-    def setUp(self):
+class BasicTransformTests():
```

Comment:
This won't run with a name like that; it should be `TestBasicTransform`.
```suggestion
class TestBasicTransform:
```